lib/file-perms.c

```diff
@@ -119,6 +119,17 @@ file_perm_options_init(FilePermOptions *self, GlobalConfig *cfg)
 }
 
 gboolean
+file_perm_options_check_file_is_device(const gchar *name)
+{
+  struct stat st;
+
+  if (stat(name, &st) >= 0)
```

Comment:
I suggest to you use `lstat` in order to handle correctly the symlinked device files.